### ISSUE TITLE
abdominal cyst

### ISSUE THREAD

**Issue by sng2140** [obophenotype/human-phenotype-ontology#7734]
**Preferred term label:**
Abdominal cyst

**Synonyms**

**Definition (free text, please give PubMed ID)**
Cyst in the abdomen 

Comment: fetal ultrasound cannot always distinguish the origin of a cystic lesion in the abdominal region. Pubmed 34189111

**Parent term (use hpo.jax.org/app)**
fetal anomaly; abnormal fetal morphology 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@sng2140
Added as HP:4000139

definition:
Fetal abdominal cysts are usually detected during the second trimester anomaly scan or discovered incidentally at later gestations. However, a definitive diagnosis is often not made until postnatal life. The most common etiologies of fetal abdominal cysts are: ovarian, gastrointestinal cystic duplication, liver and choledochal, meconium pseudocysts, mesenteric and adrenal. Following identification of a cyst, careful morphological assessment allows the prediction of their natural history and importantly, the likelihood of surgical intervention being required postnatally.
